gtk2_ardour/virtual_keyboard_window.cc

```diff
@@ -509,7 +509,7 @@ VirtualKeyboardWindow::pitch_bend_update_tooltip (int value)
 	_pitch_slider_tooltip->set_tip (string_compose (
 	      _("Pitchbend: %1\n"
 	        "Use mouse-drag for sprung mode,\n"
-	        "mouse-wheel for presisent bends.\n"
+	        "mouse-wheel for precise bends.\n"
```

Comment:
It should be *persistent*.

Mouse-wheel offsets the pitch persistently (not a sprung more).